packages/babylon/src/parser/expression.js

```diff
@@ -525,10 +525,15 @@ export default class ExpressionParser extends LValParser {
       }
       return this.finishNode(node, "MemberExpression");
     } else if (!noCalls && this.match(tt.parenL)) {
+      const oldMaybeInArrowParameters = this.state.maybeInArrowParameters;
+      const oldYOAIPAP = this.state.yieldOrAwaitInPossibleArrowParameters;
```

Comment:
Any sugestion for a shorter name than `yieldOrAwaitInPossibleArrowParameters`? :laughing:

Reply:
None other than Parameters -> Params, but not a huge difference lol